### ISSUE TITLE
arg “--frozen” in train.py make no sense？

### ISSUE THREAD

**Issue by stoneyang159** [cleardusk/3DDFA#153]
Hi，when i tried to re-train the model in this repo, I noticed that "While the strategy using VDC to finetune WPDC achieves the best result." is mentioned in your readme file. So I am curious how did your finetune WPDC model? Did you use arg of "--frozen" to make some layer with no grad? I try to write --frozen=="true" in .sh, but make no sense. 

Thanks in advance for your reply.

**Comment by cleardusk** [comment]
`frozen` is a redundancy config. Just using VDC loss function to fine-tune a trained model by WPDC is ok.

**Comment by stoneyang159** [comment]
thanks
